### PR TITLE
Access_Type definitions

### LINKED ISSUE #140: Access_Type: updated definitions
The existing Access_Type definitions have been a source of confusion, which the new definitions aim to resolve. The updated definitions are as follows:
**Controlled.** "At the time of the Request or Investigation the content item was restricted to authorized users (e.g. behind a paywall) on this platform. This includes free content that is only available to authorized (registered) users."
**Open** (was OA_Gold). "At the time of the Request or Investigation the content item was available to all users on this platform, regardless of authorization status, under an Open Access model. Open applies where the content provider asserts that the content is Open Access, irrespective of the license associated with the content item (that is, while the content item may be under a Creative Commons license this is not essential). Open content items may be in hybrid or fully Open Access publications. Open content items may have been Open from the day of publication, or after expiry of an embargo, but it is not intended to return to Controlled status."
**Free_To_Read** (newly applicable). "At the time of the Request or Investigation the content item was available to all users on this platform, regardless of authorization status, but was not Open. The content item may or may not have been Controlled at some point in the past, and may or may not return to Controlled status in the future (e.g. promotional materials where these can be tracked by the platform, or archival content a publisher has made free to read)."

Affects
- Section 3.1.2.3 Title Usage Standard Views
- Section 3.2.2 Report Body
- Section 3.3.5 Access Types
- Section 4.3 Title Reports
- Section 4.3.1 Report Header (TR)
- Section 4.3.3 Filters and Attributes (TR)
- Section 4.4.3 Filters and Attributes (IR)
- Appendix A Glossary of Terms
- Appendix E Audit Tests

### PR DESCRIPTION
**Pull request by tashalouizamc**
Updated the access_type definitions, use of access_types throughout the cop, and the terminology around standard view names

### CODE REVIEW

File: source/03-specifications/03-counter-report-common-attributes-and-elements.rst [hunk 1]
```diff
@@ -526,6 +526,11 @@ Access Types
 
 In order to track the value of usage for licensed content, librarians want to know how much Open Access or other freely available content was used and how much content was behind a paywall. To accommodate this R5 has introduced an Access_Type attribute with values of Controlled, OA_Gold, OA_Delayed, and Other_Free_To_Read. The table below lists the Access_Types and the Host_Types and reports they apply to. Note that Access_Type relates to access on the platform where the usage occurs: if access to a Gold Open Access article is restricted on a platform (for example because the article is included in an aggregated full-text database available to subscribers only) the Access_Type is Controlled.
 
+Note that the values for Access_Type changed in R5.1 to reflect community needs around reporting and to address common misunderstandings. 
+
+* Open was previously known as OA_Gold.
```
Comment (berndoberknapp) [inline source/03-specifications/03-counter-report-common-attributes-and-elements.rst]: This seems misleading to me - OA_Gold is covered by Gold, but Gold includes much more than just OA_Gold.

The paragraph above the note still lists the old Access_Types. If we want to keep that, I would suggest to add the note about the changes after that sentence, and then start a new paragraph with "The table below...".

File: source/appendices/a-glossary-of-terms.rst [hunk 2]
```diff
@@ -690,14 +690,16 @@ Note: The main Code of Practice document takes precedence in the case of any con
...
+   * - OA_Gold
+     - See Open.
      -
 
-   * - OA_Gold
-     - A COUNTER Access_Type. At the time of the transaction, the content item was available under a Gold Open Access license (content that is immediately and permanently available as Open Access because an article processing charge applies or the publication process was sponsored by a library, society, or other organization). Content items may be in hybrid publications or fully Open Access publications.
+   * - Open
+     - A COUNTER Access_Type, previously known as OA_Gold. At the time of the Request or Investigation the content item was available to all users on this platform, regardless of authorization status, under an Open Access model. Open applies where the content provider asserts that the content is Open Access, irrespective of the license associated with the content item (that is, while the content item may be under a Creative Commons license this is not essential). Open content items may be in hybrid or fully Open Access publications. Open content items may have been Open from the day of publication, or after expiry of an embargo, but it is not intended to return to Controlled status.
```
Comment (berndoberknapp) [inline source/appendices/a-glossary-of-terms.rst]: As mentioned above the "previously known as OA_Gold" seems misleading to me.

**Review by berndoberknapp — CHANGES_REQUESTED** [review]
There is a number of places where OA_Gold is still used (besides appendices which will be replaced): Section 1.2.4, caption for Figure 3.b (maybe covered by #165), above Table 3.s, Section 8.1 and Section 11.1.

The Host_Types and Reports in Table 3.u need to be updated for Free_To_Read. I assume Open (and OA_Gold in 5.0?) also would apply to eBook_Collection which is required to provide TR?

**Comment by tashalouizamc** [comment]
New commit to address comments, 2023-03-15

File: source/appendices/a-glossary-of-terms.rst [hunk 3]
```diff
@@ -690,14 +690,16 @@ Note: The main Code of Practice document takes precedence in the case of any con
      - See Open Access.
      -
 
-   * - OA_Delayed
-     - A COUNTER Access_Type that is reserved for future use and must not be implemented. At the time of the transaction, the content item was available as Open Access because the publisher’s embargo period had expired (delayed Open Access).
+   * - OA_Gold
+     - See Open.
```
Comment (berndoberknapp) [inline source/appendices/a-glossary-of-terms.rst]: Open on longer contains a reference to OA_Gold, so maybe the change should be explained here?

File: source/appendices/a-glossary-of-terms.rst [hunk 4]
```diff
@@ -729,7 +731,7 @@ Note: The main Code of Practice document takes precedence in the case of any con
      -
 
    * - Other_Free_to_Read
-     - A COUNTER Access_Type for institutional repositories. At the time of the transaction, the content item was freely available for reading (no license required) and did not qualify under the OA_Gold Access_Type.
+     - See Free_To_Read.
```
Comment (berndoberknapp) [inline source/appendices/a-glossary-of-terms.rst]: Free_To_Read on longer contains a reference to Other_Free_To_Read, so maybe the change should be explained here?

**Review by berndoberknapp — CHANGES_REQUESTED** [review]
The Host_Types and Reports in Table 3.u still need to be updated for Free_To_Read.

I assume Open (and OA_Gold in 5.0?) also would apply to eBook_Collection which is required to provide TR?

**Comment by tashalouizamc** [comment]
> The Host_Types and Reports in Table 3.u still need to be updated for Free_To_Read.
> 
> I assume Open (and OA_Gold in 5.0?) also would apply to eBook_Collection which is required to provide TR?

I left this one the other day because I was still pondering it, but I wonder why the access types need to be restricted at all - shouldn't they be for all host types?

**Comment by berndoberknapp** [comment]
> I left this one the other day because I was still pondering it, but I wonder why the access types need to be restricted at all - shouldn't they be for all host types?

The Access_Types indeed don't have to be restricted, but the tables only show what is required. Since Access_Type is only applicable for TR and IR, only the Host_Types required to provide TR or IR are listed. I've adapted the Host_Types accordingly and fixed the Reports for Free_To_Read.

**Review by berndoberknapp — APPROVED** [review]
The text in Section 3.3.5 was changed both in #171 and this pull request. I've kept the version from #171, but included the note from this pull request.

**Pull request merged**